src/components/dev-hub/card-badge.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { colorMap, size } from './theme';
+import { P5 } from './text';
+import styled from '@emotion/styled';
+
+const CardBadge = styled('div')`
+    bottom: 0;
+    background-color: ${colorMap.greyDarkThree};
+    display: inline-block;
```

Comment:
I don't think we need this actually? The absolute positioning should be taking care of our layout concerns here and it looks fine without it, were you seeing something different?

Reply:
You're right! I added this before adding absolute positioning and forgot to take it out aftwerwards.